### ISSUE TITLE
When the backend crashes half way during generation. The WebUI will delete the texts that's already been generated and streamed.

### ISSUE THREAD

**Issue by WingFoxie** [LostRuins/koboldcpp#499]
Currently, if I generate more text, and somehow the backend (I mean that cmd window) crashes during the generation process. An error will show like this one:

```
Error Encountered
Error while submitting prompt: TypeError: NetworkError when attempting to fetch resource.
```

But I had "token streaming" on! And some of the text that's already been streamed to the WebUI, will be DELETED as well??
Please don't delete them! At least provide an option to preserve them!

Currently I just have to act fast, and try to grab the text and copy them somewhere else, before it's removed.

**Comment by LostRuins** [comment]
Does this issue still happen?

**Comment by WingFoxie** [comment]
It does not happen anymore, only because I don't experience crashing anymore. 
The behavior when it crashes, stays the same.

<details>
<summary>Details:</summary>
<p>

If I intentionally close the console window when a response is being generated, simulating a "crash". It still drops whatever text that's already streamed into the WebUI. **When the error message pops out, the text is already deleted.** (Not sure why it keeps streaming for a few seconds after the "Windows Terminal" window's already gone, though.)
(Tried v1.51.1 and v1.52 . And tried Token Streaming "On" and "SSE". Same.)

It's just that I don't experience crashing anymore in the first place.
So it's a "No" for myself.
Since it was only during that few unfortunate days around the time I opened this issue, did the backend crash multiple times a day.
I don't know why it crashes multiple times a day for a few days, and I don't know how it's fixed. 
(Didn't update Windows, or display driver, or KoboldCPP during the time. Didn't even reboot, yet one day it doesn't crash anymore.)
(It can't be that the Windows Terminal update fixes it, right?)
</p>
</details>

**Comment by LostRuins** [comment]
Yeah the reason why the text disappears is because it's only saved to file when the streaming is completed or aborted.

**Comment by LostRuins** [comment]
Yeah, maybe the connection timed out. Try using SSE streaming
